packages/logs/src/boot/startLogs.ts

```diff
@@ -32,6 +33,12 @@ const LogStatusForApi = {
 
 export function startLogs(configuration: LogsConfiguration, logger: Logger) {
   const internalMonitoring = startInternalMonitoring(configuration)
+  const monitoringBatch = startMonitoringBatch(
+    configuration,
+    configuration.rumEndpointBuilder,
+    configuration.replica?.rumEndpointBuilder
+  )
+  internalMonitoring.telemetryEventObservable.subscribe((event) => monitoringBatch.add(event))
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏This got me confused, and maybe it could be clearer if `startInternalMonitoring` received the `monitoringBatch` as an argument. In would require a bit of work in in RUM because currently the batch is created only when we got the first View, but we could create it earlier. In this PR, any telemetry event produced before connecting the observable to the batch is ignored, so I think it make sense to create the batch as early as possible.

Reply:
We could have a `batchInterface` as a dependency but with the observable we don't really need to be coupled with it and we can just let the caller do the wiring.
I can experiment with trying to create RUM batch early and see how it could look like, it would probably be in a preliminary PR though.
wdyt?

Reply:
Sounds good to me, let's experiment!